src/app/game/tic-tac-toe/multi-player/page.js

```
@@ -1,254 +1,390 @@
-                {board.map((cell, index) => (
-                    <button key={index} className={styles.gridButton} onClick={() => handleClick(index)}>
+          </div>
+          <div className={styles.board} role="grid" aria-label="Tic Tac Toe board">
+            {board.map((cell, idx) => (
+              <button
+                key={idx}
+                className={styles.cell}
+                onClick={() => handleClick(idx)}
+                disabled={!!cell || !gameActive || winner || tie || (isXNext && mySymbol !== "X") || (!isXNext && mySymbol !== "O") || !bothPlayersJoined || loading}
+                aria-label={`Cell ${idx + 1} ${cell ? cell : "empty"}`}
+                role="gridcell"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<td>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)